patterns/Prototype/README.md

```diff
@@ -0,0 +1,112 @@
+# Prototype Design Pattern
+
+
+The Prototype Design Pattern is a creational pattern that enables the creation of new objects by copying an existing object. Prototype allows us to hide the complexity of making new instances from the client. The concept is to copy an existing object rather than create a new instance from scratch, something that may include costly operations. The existing object acts as a prototype and contains the state of the object.
+<br>
```

Comment:
Please remove all HTML tags, such as `<br>`. They are not necessary. If you want a space between paragraphs, just make sure there is an empty line between them in the markdown.